python/ray/util/actor_pool.py

```diff
@@ -1,7 +1,10 @@
-from typing import List, Callable, Any
+from typing import List, Callable, Any, TypeVar
 
 import ray
 from ray.util.annotations import DeveloperAPI
+import ray.actor
```

Comment:
You don't need this import

Reply:
Without I get the following in vscode/pylance:

<img width="893" alt="CleanShot 2023-06-17 at 10 11 54@2x" src="https://github.com/ray-project/ray/assets/125105/df0dbf3d-51ea-4a98-81e9-afceba8c0120">